### PR TITLE
Minor fix for explosive chemical reactions

### PR DESCRIPTION
**Pull request by N3X15**
# Changelog
:cl:
 * bugfix: Fixes a bug with explosive chemical mixtures erroneously removing holders in mobs.

### CODE REVIEW

**Comment by PJB3005** [comment]
Shitstorm in 3... 2... 1...

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -77,8 +77,12 @@
 		var/mob/living/L = holder.my_atom
 		if(L.stat != DEAD)
 			e.amount *= 0.5
-	e.start()
-	holder.clear_reagents()
+	e.start()    
+	if(istype(holder, /mob))
```
Comment (PJB3005) [inline code/modules/reagents/Chemistry-Recipes.dm]: Use `ismob`

**Review by PJB3005 — CHANGES_REQUESTED** [review]

**Review by PJB3005 — APPROVED** [review]

**Comment by ShiftyRail** [comment]
This could have used more time for discussion.
Speedmerges are (in theory) for hotfixes and game-breaking bugs.

I would also argue that the wording "minor" is debatable, and that the changelog could explicitly cite the "mechanic" being removed.

**Comment by Chazach** [comment]
that merge......
h-he's fast!

**Comment by small-boss** [comment]
are we doing pr merge speedruns too now?

**Comment by kilozombie** [comment]
Wait, so since when was this a bug?

**Comment by aceedex** [comment]
When there's peer review and discussion before merging 
🖓🕵

**Comment by ghost** [comment]
lmao nothing changes

**Comment by aceedex** [comment]
You could have stopped this

**Comment by ghost** [comment]
nobody could and nobody can except pomf and he doesn't have the balls

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 2]
```diff
@@ -77,8 +77,12 @@
 		var/mob/living/L = holder.my_atom
 		if(L.stat != DEAD)
 			e.amount *= 0.5
-	e.start()
-	holder.clear_reagents()
+	e.start()    
+	if(ismob(holder))
```
Comment (EdXbtkOEwGw) [inline code/modules/reagents/Chemistry-Recipes.dm:81]: Holy mother of snowflake batman.

If what you want to do is remove the reagent clearing (which I see no evidence of being a bug, for the record) then what is this hardcoded shit doing here? Why even differentiate mobs from other reagent holders?
The chemical reaction already implicitly removes potassium and water.

**Comment by kilozombie** [comment]
time to go back to killing and cloning
treating is for weenies :^)

**Comment by ghost** [comment]
tfw your extinguisher hit count is higher than your pill force feeds

**Pull request merged**